documentation/src/docs/getting-started/quick-start.md

```diff
@@ -0,0 +1,58 @@
+# Quick Start Guide
+
+## Download the latest Pravega
+```
+wget https://github.com/pravega/pravega/releases/download/v0.9.0-rc0/pravega-0.9.0.tgz
```

Comment:
Rather than hard coding the version maybe we could base it off of `https://github.com/pravega/pravega/releases/latest`?

Reply:
@tkaitchuck -- we have an issue with that: https://github.com/pravega/pravega/issues/5743 -- any suggestions are appreciated.

Reply:
At least, let's point to the release artifact at the moment instead of the `rc0`.